src/components/SingleArticle.jsx

```diff
@@ -28,6 +28,15 @@ const SingleArticle = (props) => {
             <h5 style={{ textAlign: "center" }}>{article.title}</h5>
           </Placeholder>
         </Grid.Row>
+        <Grid.Row centered>
+          <p
+            key={article.id}
+            id={"article-" + article.id + "-date"}
+            style={{ textAlign: "left" }}
+          >
+            Publish at: {article.published_at}
```

Comment:
I think the text should be Published at:

Reply:
yes!